src/Tribe/Tickets_View.php

```diff
@@ -1132,6 +1132,13 @@ public function get_rsvp_block( $post, $echo = true ) {
 
 		$before_content = ob_get_clean();
 
+		// Maybe echo the content from the action.
+		if ( $echo ) {
+			echo $before_content;
```

Comment:
* :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$before_content'
* :no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$before_content'.

<sub>hash: 39d003e13b1f68d90f180b29f819d0bd</sub>